persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/UserDAO.kt

```diff
@@ -18,6 +18,7 @@
 
 package com.wire.kalium.persistence.dao
 
+import com.wire.kalium.persistence.dao.ManagedByEntity.WIRE
```

Comment:
Is this import required? 🤔

Reply:
Not really, I have optimize imports enabled when doing a commit from AS, and for some reason it thought that this was an optimization